### DISCUSSION TITLE
Add an option to be able to defocus the search bar at the song select screen

### DISCUSSION THREAD

**Discussion by Newbie1st** [ppy/osu#31028] [Ideas]
### Type

Game behaviour

### Bug description

Currently it's impossible to do so, which leads to many accidental inputs and being sent to a random map, if the current one doesn't match the new filter, extremely annoying given the abyssmal performance of the song select screen, especially if you don't remember the current map's name.

### Screenshots or videos

![osu_2024-12-09_11-06-32](https://github.com/user-attachments/assets/e80aad39-21b6-43e9-8bd4-34d00f109370)
Accidental keypress
![osu_2024-12-09_11-06-37](https://github.com/user-attachments/assets/6a96d0b8-d34c-49c9-8289-927aa374bed3)
Being sent to Brazil (1 star maps)
![osu_2024-12-09_11-06-40](https://github.com/user-attachments/assets/e2ed4ee4-a0fc-4ef7-965a-a6b8950782ce)

### Version

2024.1208.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18057015/compressed-logs.zip)

**Comment by frenzibyte** [comment]
Performance concerns are already covered by https://github.com/ppy/osu/issues/29375, the rest does not constitute an issue.

**Comment by Striikerr** [comment]
I second this request. It is probably one of the few last things in Lazer that keeps me from saying, "_I objectively prefer this to stable now._" I'm not sure if it's worth opening another discussion but also having it so that it doesn't just automatically select whatever the map is that you _DO_ search for. sometimes I want to look up the name of a song, but if I don't have it, I'll have to search for wherever I was beforehand.

**Reply by peppy** [reply]
This is being fixed in an ongoing rewrite of song select.
